src/app_service/service/message/dto/removed_message.nim

```diff
@@ -7,8 +7,10 @@ include ../../../common/json_utils
 type RemovedMessageDto* = object
   chatId*: string
   messageId*: string
+  deletedBy*: string
 
 proc toRemovedMessageDto*(jsonObj: JsonNode): RemovedMessageDto =
   result = RemovedMessageDto()
   discard jsonObj.getProp("chatId", result.chatId)
   discard jsonObj.getProp("messageId", result.messageId)
+  discard jsonObj.getProp("deletedBy", result.deletedBy)
```

Comment:
Should we process empty `deletedBy` here as well?

Reply:
We don't have the original message info here, so we can't set the senderId to replace it.